app/Models/BookRepository.php

```
@@ -1038,35 +1057,62 @@ private function stringInput(mixed $value): string
-            $stmt->execute();
-            $stmt->close();
+        $del = $this->db->prepare('DELETE FROM libri_autori WHERE libro_id = ?');
+        if ($del) {
+            $del->bind_param('i', $bookId);
+            $del->execute();
+            $del->close();
         } else {
-            // If prepared statement fails, log the error and throw exception
             error_log("Critical error: Unable to prepare statement for deleting book authors for book_id: $bookId");
             throw new \Exception("Database error: unable to delete book authors");
         }
```

PHPMD (2.15.0)

[error] The method syncContributors uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)